locales/kab_DZ/LC_MESSAGES/duckduckgo.po

```diff
@@ -71,20 +71,7 @@ msgstr "%s deg %s"
 
 msgctxt "install-duckduckgo"
 msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
-msgstr ""
-
-msgctxt "install-duckduckgo"
-msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
-msgstr ""
-"%sTekki %sAllow%sTekki %sAdd%sSenqed %sAllow%s, sakin tekki ɣef %sOkay%s"
-
-msgctxt "install-duckduckgo"
-msgid ""
-"%sClick %sContinue To Installation%sClick %sAdd%sCheck %sAllow%s, then click "
-"%sOkay%s"
-msgstr ""
-"%sTekki %sKemmel tasuqqilt%sTekki %sAdd%sSennqed %sAllow%s, sakin tekki ɣef "
-"%sOkay%s"
+msgstr "%sKliki %s"Rnu%sYafhas %sSireg%s, sakin kiliki %sOk%s"
```

Comment:
It looks like you accidentally deleted a bunch of strings, plus there is a problem with that `%s"Rnu%s`. The `"` closes the string and creates a syntax error. Do you mind reviewing and getting back to me?

Reply:
I do it asap